### DISCUSSION TITLE
ButtonGroupColumn set width/class attributes

### DISCUSSION THREAD

**Discussion by howdu** [rappasoft/laravel-livewire-tables#910] [Q&A] [answered]
I want my action buttons to be as small as possible, letting the rest of the columns fill out. 

Unfortunetly ButtonGroupColumn attributes add the class to the containing `<div>` and not the `<td>`

**Comment by jadersbr** [comment]
Mabe `setTdAttributes` will help you.

**Comment by bayukartiko** [comment, marked as answer]
maybe you haven't read the documentation completely [https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/other-column-types#content-button-group-columns](https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/other-column-types#content-button-group-columns).

I think this is what you want:
```
   ButtonGroupColumn::make('Actions')
      ->attributes(function($row) {
         return [
            'class' => 'text-center',
         ];
      })
      ->buttons([])
```

or if you want the `Actions` column is small than others column, maybe you must tried this: [https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods#content-setthattributes](https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods#content-setthattributes)
```
    $this->setThAttributes(function(Column $column) {
        if ($column->getTitle() == "Actions") {
            return [
                'default' => false,
                'class' => 'text-center',
                'style' => 'width: 0%;',
           ];
        }
        return [];
    });
```

**Comment by howdu** [comment]
Thanks guys - still a little new to Livewire. I had read the documentation but forgive me for missing the part where you can use `setThAttributes` for specific columns.
